src/scripts/yast2_container

```diff
@@ -52,6 +48,18 @@ if [ "$VERSION_CHECK" != "0" ] && [ "$ID" != "opensuse-tumbleweed" ] && [ "$ID"
     exit 1
 fi
 
+if [ -z "$YCONTAINERS_SKIP_WARNING" ]; then
+  echo "** WARNING: This is an experimental tool, do not use it in production systems!!"
+  echo "** There is a risk of breaking the system or data loss!"
+  echo
+  echo "** WARNING: Using development containers which is not stable."
+  echo
+  echo "Set env variable YCONTAINERS_SKIP_WARNING to skip this warning."
+  echo
+  echo "Press enter to continue or ctrl+c to abort."
+  read -e || exit $?
```

Comment:
Um, why not just `exit 1`?  I think we do not need to expose the exact `read` error, we can hide the implementation detail.